configure.ac

```diff
@@ -79,7 +79,11 @@ AC_ARG_ENABLE(redis-test,
 esac],[rtest=false])
 AM_CONDITIONAL(RTEST, test x$rtest = xtrue)
 
-AM_CONDITIONAL(ARCH64, test `getconf LONG_BIT` = "64")
+if test x$CONFIGURED_ARCH = xarmhf && test x$CROSS_BUILD_ENVIRON = xy; then
```

Comment:
what about this one ? CROSS_BUILD_ENVIRON ?

Reply:
I don't know exactly how to fix it. We can't run getconf on host machine to determine ARCH64 of the target machine. Do you want me to pass ARCH64 env. variable and check if it is already defined then do nothing otherwise test getconf ? Any other suggestion ?

Reply:
im not that familiar with proper way to pass cross compiling flags and whether that approach will be suitable for sonic, are you the only one working on this? maybe this would require some broader audience ?
i found this example for kernel https://www.centennialsoftwaresolutions.com/post/stop-typing-make-arch-arm-cross_compile-arm-linux-gnueabi-o-when-building-the-linux-kernel

Reply:
I'm working alone, but Antony Rheneus is helping me. What specifically is not good in the current confugure.ac code ? That CONFIGURED_ARCH and CROSS_BUILD_ENVIRON variables from Sonic are exposed ? That it is not generic solution for cross-compilation ?

Reply:
nothing wrong, just wondering that there should be already established some common way to crosscompile in makefiles